echopype/echodata/echodata.py

```diff
@@ -256,6 +256,42 @@ def __setattr__(self, __name: str, __value: Any) -> None:
                         self._tree[group_path].ds = __value
         super().__setattr__(__name, attr_value)
 
+    def _harmonize_env_param_time(
+        self, p: Union[int, float, xr.DataArray], ping_time: Union[xr.DataArray, datetime.datetime]
```

Comment:
```suggestion
        self, p: Union[int, float, xr.DataArray], ping_time: Union[xr.DataArray, datetime.datetime] = None
```

This change shouldn't have any effect, but it'll make it clearer that `ping_time` is in effect optional. It's not used (and therefore not passed) for EK60. Maybe you can add a test for `ping_time is not None` below, before the statement `p_new = p.interp(time1=ping_time)`

Reply:
Yes, thanks for catching this. I did the typing and forgot to add back the =None part.

And good point for testing for `ping_time` -- this interp part is not used at all in the current code (since our time1 is always length=1 from EK80), but it will surely come in later once we combine echodata. I will add a test for this too, to make sure the operations are done correctly (although it is arguable whether we should do `method="linear"` or `method="nearest"` or something else).

Reply:
I decided to use `method="linear"` and remove the check for `sonar_model` so that the function is general. I realized that even for EK60 it could be that users provide data arrays as inputs of the `env_params`, and in that case we can't just do the `time1` to `ping_time` renaming either.

Reply:
Done!